Source/carthage/Build.swift

```diff
@@ -207,38 +207,14 @@ the signing identities available
 public func buildableSDKs(sdks: [SDK], _ scheme: String, _ configuration: String, _ project: ProjectLocator, _ formatting: ColorOptions.Formatting) -> SignalProducer<[SDK], CarthageError> {
 	var identityCheckArgs = BuildArguments(project: project, scheme: scheme, configuration: configuration)
 	
-	return parseSecuritySigningIdentities()
```

Comment:
@NachoSoto want me to remove def + tests that refer to `parseSecuritySigningIdentities`?